src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -71,8 +71,19 @@ public class DAOTestHelper extends AbstractTestHelper {
   @SuppressWarnings("rawtypes")
   private static PostgreSQLContainer postgresContainer;
 
-  @BeforeAll
-  public static void startUp() throws Exception {
+   @Override
+  public void testPlanExecutionStarted(TestPlan testPlan) {
+     if (disableTestContainers()) {
```

Comment:
If we don't need containers, we're not starting the app so we can just return here.

Reply:
This is an important distinction - it would be nice to have some kind of commentary here. In other words, `disableTestContainers` indicates two things:
1. Test containers are not necessary
2. App startup is not possible because it relies on existing test containers (postgres + mockserver).